mozillavpn_it.xlf

```diff
@@ -189,70 +200,80 @@
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">46</context></context-group>
         <context-group purpose="location"><context context-type="linenumber">83</context></context-group>
         <context-group purpose="location"><context context-type="linenumber">279</context></context-group>
       </trans-unit>
-      <trans-unit id="_msg31">
+      <trans-unit id="_msg33">
         <source xml:space="preserve">Connecting…</source>
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">116</context></context-group>
       </trans-unit>
-      <trans-unit id="_msg32">
+      <trans-unit id="_msg34">
         <source xml:space="preserve">Masking connection and location</source>
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">121</context></context-group>
       </trans-unit>
-      <trans-unit id="_msg33">
+      <trans-unit id="_msg35">
         <source xml:space="preserve">VPN is on</source>
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">158</context></context-group>
       </trans-unit>
-      <trans-unit id="_msg34">
+      <trans-unit id="_msg36">
         <source xml:space="preserve">Secure and private  •  </source>
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">163</context></context-group>
       </trans-unit>
-      <trans-unit id="_msg35">
+      <trans-unit id="_msg37">
         <source xml:space="preserve">Disconnecting…</source>
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">195</context></context-group>
       </trans-unit>
-      <trans-unit id="_msg36">
+      <trans-unit id="_msg38">
         <source xml:space="preserve">Unmasking connection and location</source>
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">200</context></context-group>
       </trans-unit>
-      <trans-unit id="_msg37">
+      <trans-unit id="_msg39">
         <source xml:space="preserve">Switching…</source>
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">233</context></context-group>
       </trans-unit>
-      <trans-unit id="_msg38">
+      <trans-unit id="_msg40">
         <source xml:space="preserve">From %1 to %2</source>
         <target xml:space="preserve"></target>
         <context-group purpose="location"><context context-type="linenumber">238</context></context-group>
       </trans-unit>
+      <trans-unit id="_msg41">
+        <source xml:space="preserve">Connection Information</source>
+        <target xml:space="preserve"></target>
```

Comment:
We shouldn't have an empty target if the string is not translated. Not sure if it's just the result of testing though.